modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -0,0 +1,169 @@
+                        "3.223.147.2/32",
+                        "54.88.225.66/32",
+                        "54.177.143.128/32",
+                        "54.219.250.189/32",
+                        "18.135.14.84/32",
+                        "18.135.218.119/32",
+                        "3.11.51.55/32",
+                        "3.233.86.181/32",
+                        "34.226.229.129/32",
+                        "44.194.44.138/32",
+                        "34.216.5.118/32",
+                        "52.11.52.9/32",
+                        "52.40.203.77/32",
+                        "18.217.173.123/32",
+                        "3.13.34.119/32",
+                        "3.19.160.232/32",
+                        "18.188.187.143/32",
+                        "18.216.245.132/32",
+                        "52.14.85.89/32",
+                        "52.21.184.148/32",
+                        "44.194.57.131/32",
+                        "18.188.65.148/32",
+                        "3.130.101.176/32",
+                        "3.130.198.233/32",
+                        "54.210.128.71/32",
+                        "54.227.100.14/32",
+                        "54.92.188.93/32",
+                        "107.22.5.187/32",
+                        "3.217.212.27/32",
+                        "52.22.56.3/32",
+                        "52.5.10.152/32",
+                        "54.237.41.201/32",
+                        "34.202.145.72/32",
+                        "52.205.239.95/32",
+                        "54.236.208.68/32",
+                        "3.234.64.191/32",
+                        "34.195.159.252/32",
+                        "34.228.34.122/32",
+                        "54.205.89.242/32",
+                        "209.132.0.0/16",
+                        "66.187.0.0/16",
+                        "2620:0052:0004:0000:0000:0000:0000:0000/48"    
+                    ]
+                },
+                "Bool": {
+                    "aws:ViaAWSService": "false"
+                }
+            }
+        }
+    ]
+}
+----
++
+[NOTE]
+====
+This list is subject to change. Additionally, you must specify the IP addresses in CIDR notation.  
+====
++
+. In the Create policy page, click *Review and create*.
+. In the Review and create page, provide a name and description for the policy, and review the details to ensure accuracy.
+. Click *Create policy* to save the policy.
+
+[NOTE]
+====
+The condition key `aws:ViaAWSService` must be set to false for enabling subsequent calls to succeed on behalf of the initial call. For example, if you were to make a call to `aws ec2 describe-instances`, any subsequent calls made within the AWS API server to retrieve information about the EBS volumes attached to the ec2 instance would fail if condition key `aws:ViaAWSService` is not set to false. This is because these subsequent calls would originate from AWS IP addresses, which are not included in the AllowList.
```

Comment:
```suggestion
The condition key `aws:ViaAWSService` must be set to false to enable subsequent calls to succeed based on the initial call. For example, if you make an initial call to `aws ec2 describe-instances`, all subsequent calls made within the AWS API server to retrieve information about the EBS volumes attached to the ec2 instance will fail if the condition key `aws:ViaAWSService` is not set to false. The subsequent calls would fail because they would originate from AWS IP addresses, which are not included in the AllowList.
```